package.json

```diff
@@ -43,15 +43,15 @@
     "test:git": "git diff --exit-code",
     "release": "npm run build",
     "pack": "turbo run pack",
-    "publish-packages": "turbo run clean && rimraf .turbo && npm run install-ws && turbo run build lint test",
+    "publish-packages": "npm run install-ws && turbo run build lint test",
     "versionup:patch": "turbo run version --no-git-tag-version patch",
     "versionup:minor": "turbo run version --no-git-tag-version minor",
     "versionup:major": "turbo run version --no-git-tag-version major",
     "prepare": "husky install",
     "test:ci": "if test \"$NODE_ENV\" = \"production\"; then echo \"Yes!\"; fi"
   },
   "dependencies": {
-    "@amzn/iot-black-pearl-internal-v3": "file:./iot-black-pearl-internal-v3.tgz"
+    "@aws-sdk/client-iotsitewise": "^3.696.0"
   },
```

Comment:
This should be included in each package using it and not at the root. It's a mistake when we put non-repo-wide devDependencies at the root. See https://turbo.build/repo/docs/crafting-your-repository/managing-dependencies#install-dependencies-where-theyre-used.